### ISSUE TITLE
Mods won't deploy for Ready Or Not

### ISSUE THREAD

**Issue by zzzombine** [Nexus-Mods/Vortex#14180]
**Describe the bug**
When I try to deploy mods for Ready Or Not, Vortex does nothing. The message window about deploying shows up and nothing more. And I don't get any error messages.

**To Reproduce**
1. Go to 'Mods' menu
2. Click on 'Deploy Mods'
3. Mods doesn't deploy

**Expected behavior**
I expected it to deploy mods as usual. Usually after deploy .pak files created in the directory of the game.

**Platform (please complete the following information):**
 - OS: Windows 10
 - Vortex 1.7.8 (Downloaded from GitHub)

**Additional context**
This problem occurred after I verified files of the game in Steam, because I didn't had a couple of audio files. And after verification, I replaced some audio files with modded. I think that my problem is unique, because I did not find similar problems on the Internet.

Files:
[vortex.log](https://github.com/Nexus-Mods/Vortex/files/11093379/vortex.log)
I did not find 'state' folder and the 'state' folder in the game folder

**Comment by TanninOne** [comment]
This seems to be caused by the recent update to the "Unreal Engine Mod Installer" extension @Pickysaurus 
Here's the error messages that get written to the log (excerpt):
```
Tue, 28 Mar 2023 19:06:03 GMT - error: [mod-dependency-manager] invalid file entry - what is this? filePath=F:\Vortex Mods\readyornot\1. Surefire Flashlight 1.2 - ADAM-1458-1-2-1670538730\pakchunk99-Mods_Surefire_Flashlights_1200Lumens_P.pak, isDirectory=false, isReparsePoint=false, size=53248, mtime=1670537985, isTerminator=false, error=TypeError: game.details.unrealModsPath is not a function
    at Object.getPath (C:\Users\Ilya\AppData\Roaming\Vortex\plugins\Vortex Extension Update - Unreal Engine Mod Installer v1.1.2\index.js:51:96)
    at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\mod-dependency-manager\index.js:2:654419
    at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\mod-dependency-manager\index.js:2:654836
    at Array.forEach (<anonymous>)
    at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\mod-dependency-manager\index.js:2:654724
    at Object.onProgress (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\turbowalk\index.js:30:11)
Tue, 28 Mar 2023 19:06:03 GMT - error: [mod-dependency-manager] invalid file entry - what is this? filePath=F:\Vortex Mods\readyornot\30 Slots-1268-1-0-1655766442\pakchunk99-More_Ammo_30_Slots_P.pak, isDirectory=false, isReparsePoint=false, size=16384, mtime=1655730232, isTerminator=false, error=TypeError: game.details.unrealModsPath is not a function
    at Object.getPath (C:\Users\Ilya\AppData\Roaming\Vortex\plugins\Vortex Extension Update - Unreal Engine Mod Installer v1.1.2\index.js:51:96)
    at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\mod-dependency-manager\index.js:2:654419
    at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\mod-dependency-manager\index.js:2:654836
    at Array.forEach (<anonymous>)
    at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\mod-dependency-manager\index.js:2:654724
    at Object.onProgress (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\turbowalk\index.js:30:11)
Tue, 28 Mar 2023 19:06:03 GMT - error: [mod-dependency-manager] invalid file entry - what is this? filePath=F:\Vortex Mods\readyornot\AFG-2 - MLOK-1996-1-0-1668790132\pakchunk999-Mods_BAL_AFG-MLOK_P.pak, isDirectory=false, isReparsePoint=false, size=77770752, mtime=1668188713, isTerminator=false, error=TypeError: game.details.unrealModsPath is not a function
    at Object.getPath (C:\Users\Ilya\AppData\Roaming\Vortex\plugins\Vortex Extension Update - Unreal Engine Mod Installer v1.1.2\index.js:51:96)
    at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\mod-dependency-manager\index.js:2:654419
    at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\mod-dependency-manager\index.js:2:654836
    at Array.forEach (<anonymous>)
    at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\mod-dependency-manager\index.js:2:654724
    at Object.onProgress (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\turbowalk\index.js:30:11)
```

**Comment by TanninOne** [comment]
should be fixed with version 1.1.3 of the extension
